sdk/extensions/Microsoft.Extensions.Azure/tests/ClientFactoryTests.cs

```diff
@@ -204,7 +204,7 @@ public void CreatesCertificateCredentialsAdditionalTenants(string additionalTena
             Assert.AreEqual("ConfigurationTenantId", clientCertificateCredential.TenantId);
 
             var actualTenants = (string[]) typeof(ClientCertificateCredential)
-                .GetField("_additionallyAllowedTenantIds", BindingFlags.NonPublic | BindingFlags.Instance)
+                .GetFields(BindingFlags.NonPublic | BindingFlags.Instance).First(f => f.Name.EndsWith("dditionallyAllowedTenantIds"))
```

Comment:
Why not use the full word 😆

Reply:
The casing?

Reply:
ust ecause!

Reply:
😆 
After I left the comment I realized this may be intentional in case we switch between casing.

Reply:
> 😆 After I left the comment I realized this may be intentional in case we switch between casing.

Yeah - the name will be either `_additional...` or `Additional...`

Reply:
I had the same thought initially, but realized this would solve the problem in the short window where a PackageReference or ProjectReference could make a difference.